### ISSUE TITLE
supported Emmet functions

### ISSUE THREAD

**Issue by rcrooks** [osxpeppermint/peppermint#78]
I know that I can expand abbreviations with a tab, but are there other Emmet functions supported, and if so, how are they accessed? Would be good to have this in the menus or at least documentation at some point.

**Comment by osxpeppermint** [comment]
Well, Emmet functions are there, but they're not part of the snippets system (you're not going to find them in the autocompletion menu - since basically they're not exactly a "snippet" auto-completion).

Now, here's the catch (though I'll have to find a way to sort it out - perhaps with a Preferences setting?):

When the autocompletion menu pops up, and you hit <key>Tab</key>, the first item is automatically selected (be it a keyword, snippet, anything...).

Now, if you want to make use of pure Emmet (and not some of its snippet-ized variations), the autocompletions drop down should be off. And then you just hit Tab.

Right now, there is no option to disable automatic completions. However, what you can do is hit "Esc" in order to close the automatic completion and *then* hit Tab. Emmet will be there :-)

Please have a look at the example below:

![emmet](https://cloud.githubusercontent.com/assets/9476915/8271868/fc870176-182d-11e5-8f6e-bfcf5688b8b4.gif)

**Comment by rcrooks** [comment]
Right, I got that. I was wondering if you can also do things like wrap a selection with an abbreviation, though, as well as just expanding abbreviations. Some other useful Emmet functions I’ve seen in some implementations:
balance (inward or outward)
update image size
increment or decrement numbers

-- 
Robert Crooks
Director of Learning Services

On June 21, 2015 at 9:56:16 AM, Peppermint - Code Editor for Mac (notifications@github.com) wrote:

Well, Emmet functions are there, but they're not part of the snippets system (you're not going to find them in the autocompletion menu - since basically they're not exactly a "snippet" auto-completion).

Now, here's the catch (though I'll have to find a way to sort it out - perhaps with a Preferences setting?):

When the autocompletion menu pops up, and you hit Tab, the first item is automatically selected (be it a keyword, snippet, anything...).

Now, if you want to make use of pure Emmet (and not some of its snippet-ized variations), the autocompletions drop down should be off. And then you just hit Tab.

Right now, there is no option to disable automatic completions. However, what you can do is hit "Esc" in order to close the automatic completion and then hit Tab. Emmet will be there :-)

Please have a look at the example below:

**Comment by osxpeppermint** [comment]
Hmm... I'll have to have a look into it to see what you mean (believe it or not, I'm not such a passionate Emmet user myself :) )
